site-service/src/main/resources/bootstrap.yml

```diff
@@ -0,0 +1,23 @@
+spring:
+  profiles: default
+  cloud:
+    config:
+      # If cannot connect to config server, don't start the application
+      fail-fast: true
+      discovery:
+        enabled: true
+        service-id: CONFIG-SERVER
+      retry:
+        initialInterval: 2000
+        multiplier: 1.5
+        maxInterval: 60000
+        maxAttempts: 100
+      label: main
```

Comment:
same comment regarding thevalues and the label

Reply:
https://stackoverflow.com/questions/54670804/spring-discovery-first-bootstrap-doesnt-find-configserver
This problem occurred on localhost. Decided with @eladiw to try on production first without these and if the problem happens again we can add them.
Removed label from eveywhere